src/GitHubCreateMergePRs/config.xml

```diff
@@ -1,22 +1,20 @@
 <config>
   <repo owner="dotnet" name="roslyn" updateExistingPr="true">
+    <!-- Roslyn servicing branches -->
+    <merge from="dev15.9.x" to="dev16.0-vs-deps" />
+    <merge from="dev16.0-vs-deps" to="release/dev16.3" />  
```

Comment:
what is dev16.0-vs-deps? it seems like that branch shouldn't exist if we have a requirement for our contributors to use dev16.

Reply:
There is no dev16.0 branch anymore